Library/Formula/joshua-v.rb

```diff
@@ -0,0 +1,55 @@
+class JoshuaV < Formula
+  desc "Joshua is an open-source statistical machine translation decoder."
+  homepage "http://joshua-decoder.org/"
+  url "http://cs.jhu.edu/~post/files/joshua-v6.0.4.tgz"
+  sha256 "0cf6674dc73f30e49a22179e90182492f8e8eea46710b505edb9acfb4d953d9a" 
```

Comment:
We use lowercased checksums; what I mean is that the checksum you wrote is different from the one I got:

`0cf6674dc73f30e49a22179e90182492f8e8eea46710b505edb9acfb4d953d9a` v.s.
`0e3b08c80303106599aa404bbba5c801cc5542779bef828d35ff80288190c273`

This is also the one the bot gets:
```
==> Downloading http://cs.jhu.edu/~post/files/joshua-v6.0.4.tgz
Downloaded to: /Library/Caches/Homebrew/joshua-v-6.0.4.tgz
SHA1: d240cc2b09b01feba03f0ac5d46b4c8d81ce01b1
SHA256: 0e3b08c80303106599aa404bbba5c801cc5542779bef828d35ff80288190c273
```

Are you behind a proxy that might modify HTTP responses?